### ISSUE TITLE
What wrong with input and output ?

### ISSUE THREAD

**Issue by rulonboev** [SciSharp/Keras.NET#63]
I am running next code:
                var input = new Input(new Keras.Shape(64, 12)); 
                var a = new CuDNNLSTM(32); 
                a.Set(input); 
                var output = new Dense(1, activation: "sigmoid"); 
                output.Set(a);
                var model = new Keras.Models.Model(new Input[] { input }, new BaseLayer[] { output });

but getting error:
{"ValueError : Output tensors to a Model must be the output of a Keras `Layer` (thus holding past layer metadata). Found: <keras.layers.core.Dense object at 0x0000021225394A88>"}

What wrong with that ?

**Comment by nanoxas** [comment]
I'm having the same problem. How did you fix it?
